modules/host/negotiate.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/NebulousLabs/Sia/modules"
 )
 
+// acceptNegotiation writes the 'accept' response to conn.
+func acceptNegotiation(conn net.Conn) error {
+	return encoding.WriteObject(conn, modules.AcceptResponse)
+}
```

Comment:
also, we could add an error/panic to rejectNegotiation if you try to write an error that is too large.